index.js

```diff
@@ -46,6 +49,8 @@ module.exports = async (input = 'patch', options) => {
 	const pkg = util.readPkg();
 	const pkgManager = options.yarn === true ? 'yarn' : 'npm';
 	const pkgManagerName = options.yarn === true ? 'Yarn' : 'npm';
+	const rootDir = await pkgDir(__dirname);
```

Comment:
It's not correct to use `__dirname` here as it will then try to find the package-lock.json in the `np` project and not the user's current directory. You should also use `pkgDir.sync` for speed and consistency. There's no benefit in using async in this case.

Reply:
Using `__dirname` a silly mistake...